t/06-balance.t

```diff
@@ -10,4 +10,15 @@ subtest "Balance for the Stripe marketplace" => sub {
         or diag explain $bal;
 };
 
+subtest "Balance history" => sub {
+    my $bal = stripe->get_balance_history;
+    cmp_deeply $bal, TD->superhashof({
+        data => TD->superbagof(
+            TD->superhashof({
+                object => 'balance_transaction',
```

Comment:
oh @comstud this might have just been lazy testing on my part... it's likely this is returning other objects too. let me check that